benchmark/bench/server-stress.js

```diff
@@ -27,9 +27,11 @@ export async function run(projectDir, outputFile) {
 	});
 
 	console.log('Previewing...');
-	const previewProcess = execaCommand(`${astroBin} preview --port ${port}`, {
-		cwd: root,
-		stdio: 'inherit',
+	const previewProcess = exec([`${astroBin} preview --port ${port}`], {
```

Comment:
Not sure how this one slipped away while refactoring to tinyexec. I think the correct usage here is something like this:
```suggestion
	const previewProcess = exec(astroBin, ['preview', '--port', port], {
```